pkg/gui/services/custom_commands/handler_creator.go

```diff
@@ -187,10 +189,19 @@ func (self *HandlerCreator) finalHandler(customCommand config.CustomCommand, ses
 		if customCommand.Stream {
 			cmdObj.StreamOutput()
 		}
-		err := cmdObj.Run()
+		output, err := cmdObj.RunWithOutput()
 		if err != nil {
 			return self.c.Error(err)
 		}
+		if strings.TrimSpace(output) == "" {
```

Comment:
nitpick: but could we move this if statement into the if statement below, giving that it's only relevant there